src/main/java/org/apache/commons/lang3/annotation/ToEqualsExclude.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.commons.lang3.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Use this annotation to builds a equals excluding the annotated field.
+ */
+@Retention(RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ToEqualsExclude {
```

Comment:
@britter I will go change the code for adapt to your comments.
About the ToEqualsExclude I think make sense stay outside EqualsBuilder because when you use the apache commons you need note the classes with  @ToEqualsExclude if this is inside EqualsBuilder you need note with @EqualsBuild.ToEqualsExclude. What do you think?